go-controller/pkg/cni/ovs.go

```diff
@@ -316,7 +317,9 @@ func waitForPodInterface(ctx context.Context, ifInfo *PodInterfaceInfo,
 			}
 
 			// try again later
-			time.Sleep(200 * time.Millisecond)
+			waitTime := 200 * time.Millisecond
+			time.Sleep(waitTime)
+			metrics.MetricOvsInterfaceUpWait.Add(float64(waitTime) / float64(time.Second))
```

Comment:
You can do `waitTime.Seconds()` instead of `float64(waitTime) / float64(time.Second)`

Reply:
oh! yes, done.